### PR TITLE
Add IsGroup to PlayerInfo

### PR DESCRIPTION
**Pull request by ScipioWright**
I did not test this because I haven't gone through the process of building it before. It looks like the one above it so it probably works fine.

### CODE REVIEW

File: Archipelago.MultiClient.Net/Helpers/PlayerHelper.cs [hunk 1]
```diff
@@ -291,6 +291,17 @@ public IEnumerable<PlayerInfo> GetGroupMembers(IPlayerHelper playerHelper)
...
 
+  		/// <summary>
+		/// If this player is a group, returns true.
+		/// </summary>
+		/// <returns>Returns whether the player is a group</returns>
+		public bool IsGroup()
+		{
+			if (GroupMembers != null && GroupMembers.Length > 0)
+				return true;
+    			return false;
```
Comment (BadMagic100) [inline Archipelago.MultiClient.Net/Helpers/PlayerHelper.cs:302]:
```suggestion
			return GroupMembers != null && GroupMembers.Length > 0;
```

**Review by BadMagic100 — COMMENTED** [review]
On principle a unit test for this would be good, however it's trivial enough that it's probably fine.

**Comment by BadMagic100** [comment]
Oh also this could and probably should be a property rather than a method

**Comment by ScipioWright** [comment]
Sorry, not familiar enough with the syntax. Would that be something like this?
```csharp
		public bool IsGroup { get; set; } = GroupMembers != null && GroupMembers.Length > 0;
```

**Comment by BadMagic100** [comment]
```cs
public bool IsGroup => GroupMembers != null && GroupMembers.Length > 0;
```
Above would be the best way to go, read-only property with an expression body syntax

**Comment by BadMagic100** [comment]
For educational purposes yours would allow the outside world to set it, and additionally it would be set when the instance is created instead of evaluated on request.

The most verbose correct way would be like this with a number of more concise intermediate options.
```cs
public bool IsGroup
{
    get
    {
        return GroupMembers != null && GroupMembers.Length > 0;
    }
}
```

**Review by BadMagic100 — APPROVED** [review]

**Pull request merged**